File: 프로그래머스/0/120817. 배열의 평균값/배열의 평균값.java
class Solution {
    public double solution(int[] numbers) {
        // 배열의 모든 원소를 더하기 위한 변수 선언
        int sum = 0;
        
        // 배열의 모든 원소를 더함
        for (int i = 0; i < numbers.length; i++) {
            sum += numbers[i];
        }
        
        // 배열의 평균 계산
        double average = (double) sum / numbers.length;
        
        return average; // 평균값 반환
    }
}

File: 프로그래머스/0/120817. 배열의 평균값/배열의 평균값.c
#include <stdio.h>

double solution(int numbers[], int length) {
    int totalSum = 0;

    // 배열의 모든 요소의 합을 계산
    for (int i = 0; i < length; i++) {
        totalSum += numbers[i];
    }

    // 총합을 요소의 개수로 나누어 평균 계산
    double average = (double)totalSum / length;
    
    return average;
}